### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by Tejinder15**
# Created Modal Component

## CSB Link
https://fxtgu.csb.app/Components/Modal/modal.html

modal

### CODE REVIEW

File: Components/Modal/modal.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  body.style.overflowY = "hidden";
+});
+
+// document.addEventListener("click", (e) => {
+//     if (e.target.id === "close-modal") {
+//         modalContainer.classList.remove("show-container");
+//         body.style.overflowY = "visible";
+//   }
+// });
+
```
Comment (ajitpsakri) [inline Components/Modal/modal.js:18]: You can just remove this.

**Review by NagaSatyaGeeth — APPROVED** [review]

File: Components/Modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <div class="content">
+            <h2 class="content-title">Modal</h2>
+            <p class="content-brief">
+                The Modal Component provides a solid foundation for creating dialogs, popovers and lightboxes.
+            </p>
+            <hr>
+            <div class="content-usage">
+                <h3>Simple Modal</h3>
+                <p>Click on the cross button or anywhere outside the modal to close it.</p>
+                <button id="open-modal">Modal Btn</button>
```
Comment (ajitpsakri) [inline Components/Modal/modal.html:55]: You can also use the button component class here.

File: Components/Modal/modal.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const modalContainer = document.querySelector(".modal-container");
+const body = document.querySelector("body");
+const closebtn = modal.querySelector('#close-modal');
+openModal.addEventListener("click", () => {
+  modalContainer.classList.add("show-container");
+  modal.classList.add("show");
+  body.style.overflowY = "hidden";
+});
+
+// document.addEventListener("click", (e) => {
```
Comment (RakshaPawar108) [inline Components/Modal/modal.js:12]: Remove commented code if not required to keep the codebase clean

**Review by RakshaPawar108 — APPROVED** [review]
Only one minor change suggested. Other than that looks good to merge.

**Review by ajitpsakri — CHANGES_REQUESTED** [review]
When I scroll down and click the button. the brightness of the background doesn't cover the full viewpoint. I don't know if it's feature or a have to be fixed. Refer screenshot below
image

**Pull request merged**